pipeline-modules/app-service/azure/pipeline-config.yaml

```diff
@@ -20,6 +20,11 @@ inputs:
       description: Keeps all secret variables available to pipeline.
       type: string
       default: ""
+    pool_name:
```

Comment:
Given this variable isn't internal, I'm not sure I understand how someone knows they need to use it?

Reply:
its in if statement, if someone provides a pool name as template input it will use self hosted pool else it will use Microsoft hosted pools

Reply:
@dshelley66, This came out of the group discussion in the architecture meeting last week. In case of RSAF, we believe the requirement would be that, the fortify setup along with all the licences installed would be available in a certain custom pool (maintained by services team probably). And so the user in this case would need an option to use a pre-existing pool to run the pipeline on.

In future, (we haven't talk abut  this) my speculation is, we may develop a way in CodePipes for user to generate an Azure DO pool, or AWS AMI (base image), or GCP docker image to use in the integration pipeline.